vuepress/docs/next/tutorials/create/pb/export-bundle-from-application.md

```diff
@@ -75,20 +82,20 @@ mkdir testBundle; cd testBundle
 ```
 
 ### Run the Bundler
-1. Create a child directory to hold the bundler output. Using the name `bundle` allows you to easily use the `ent prj` command with this bundle.
+1. Create a child directory to store the bundler output. Choosing the name `bundle` allows you to easily use the `ent prj` command with this bundle.
 ``` sh
 mkdir bundle
 ```
-2. Run the bundler command with your preferred settings. 
+2. Run the bundler command with your preferred settings:
 ``` sh
 ent bundler from-env --location bundle --code my-test-bundle --description “My Test Bundle”
 ```
-The bundler will inspect the application using the Entando APIs, collect information about the individual components, construct the appropriate descriptor files, and finally assemble the top-level descriptor file.
+The bundler will inspect the application using the Entando APIs, collect information about the individual components, construct the appropriate descriptor files and assemble the top-level descriptor file.
```

Comment:
take out 'the's >> inspect the application using Entando APIs, collect information about individual components, construct appropriate descriptor files